h2o-genmodel-extensions/jgrapht/src/main/java/hex/genmodel/tools/JgraphtPrintMojo.java

```diff
@@ -42,18 +42,28 @@ public void run() throws Exception {
             }
         } 
     }
+    
+    public void runFromFlow(SharedTreeGraph g, String outputPath){
+        try {
+            destination = outputPath;
```

Comment:
I dont like this side-effect, maybe add a constructor that will initialize this property

Reply:
now I am initializing in constructor and calling printPng() from run()